readme.md

```diff
@@ -40,124 +40,129 @@ $ npm -v
 
 * **Your Answer:**
 
----
+_forking is just a request for cloning_
 
 - [ ] Run `npm init -y` from the command line
 
 * **Question:** What does `npm init` do? How does the `-y` flag modify that command?
 
 * **Your Answer:**
 
----
+_creating package.json and -y saying yes to everything_
 
 - [ ] Take a look at the file that was generated by the previous command
 
 * **Question:** What is the purpose of the following keys? "name", "scripts", "license"
 
 * **Your Answer:**
-
----
+  _scripts: scripts are written as usual JSON key-value pairs where the key is the name of the script and the value contains the script you want to execute._
+  _names: It's just the name of the project._
+  _Licenses: you'll need to license it so that others are free to use, change, and distribute the software._
 
 - [ ] Create a `.gitignore` file
 
 * **Question:** What is the purpose of the `.gitignore` file? What is the significance of a "dot-file?"
 
 * **Your Answer:**
 
----
+_file that we don't want to keep trak of them in the reposetori_
+_It works anywhere, but its primary use is to hide configuration files in the home directory_
 
 - [ ] Create an `index.js` file with the following contents: `console.log('Hello, Node!')`
 
 * **Question:** From the command line, how can you run this file?
 
 * **Your Answer:**
 
----
+_node index.js_
 
 - [ ] Run `npm test` from the command line
 
-* **Question:** What happens and how is this related to what is in the `package.json` file? 
+* **Question:** What happens and how is this related to what is in the `package.json` file?
 
 * **Your Answer:**
 
----
+_result is "Error: no test specified"_
```

Comment:
Correct, but the question is about why that is. This is due to the fact that the value for "test" is a command that will print some text to the terminal and then exit.